sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <NoWarn>NU1903</NoWarn>
```

Comment:
what is NU1903?

The bulk of your changes are in the Distro project. I wouldn't expect a new error to be suppressed in the Test project.

Reply:
Local changes slipped through. I will have this removed.